eng/source-build-patches/0017-Rename-IReadOnlySet-and-ReferenceEqualityComparer-Ro.patch

```diff
@@ -0,0 +1,1715 @@
+ .../Core/Portable/Serialization/ObjectWriter.cs    |  2 +-
+ src/Compilers/Test/Core/Assert/AssertXml.cs        |  4 +--
+ .../VisualBasic/Portable/Binding/Binder_Query.vb   |  6 ++--
+ .../Portable/Binding/MemberSemanticModel.vb        |  2 +-
+ .../Portable/Compilation/MethodCompiler.vb         |  4 +--
+ .../Portable/Compilation/TypeCompilationState.vb   |  6 ++--
+ .../EditAndContinue/VisualBasicSymbolMatcher.vb    |  8 ++---
+ .../Portable/Emit/NoPia/EmbeddedType.vb            |  2 +-
+ .../Portable/Emit/NoPia/EmbeddedTypesManager.vb    |  4 +--
+ .../VisualBasic/Portable/Emit/PEModuleBuilder.vb   |  2 +-
+ .../VisualBasic/Portable/Emit/SymbolTranslator.vb  |  2 +-
+ .../AsyncRewriter.AsyncMethodToClassRewriter.vb    |  2 +-
+ .../AsyncRewriter.SpillFieldAllocator.vb           |  2 +-
+ .../DiagnosticsPass_ExpressionLambdas.vb           |  2 +-
+ ...eratorRewriter.IteratorMethodToClassRewriter.vb |  2 +-
+ .../LambdaRewriter/LambdaRewriter.Analysis.vb      | 14 ++++-----
+ .../Lowering/LocalRewriter/LocalRewriter.vb        |  4 +--
+ .../LocalRewriter_LocalDeclaration.vb              |  2 +-
+ ...neRewriter.StateMachineMethodToClassRewriter.vb |  4 +--
+ .../StateMachineRewriter/StateMachineRewriter.vb   |  2 +-
+ .../Operations/VisualBasicOperationFactory.vb      |  2 +-
+ .../Parser/BlockContexts/CompilationUnitContext.vb |  6 ++--
+ .../Portable/Semantics/OverloadResolution.vb       |  2 +-
+ .../TypeInference/TypeArgumentInference.vb         |  2 +-
+ .../EmbeddedSymbolManager.SymbolsCollection.vb     |  4 +--
+ .../Source/SourceMemberContainerTypeSymbol.vb      |  6 ++--
+ .../Retargeting/RetargetingNamedTypeSymbol.cs      |  2 +-
+ .../Portable/Symbols/Source/SourceModuleSymbol.vb  |  2 +-
+ .../Portable/Symbols/Tuples/TupleTypeSymbol.vb     |  4 +--
+ .../Symbol/SymbolsTests/InstantiatingGenerics.vb   |  8 ++---
+ .../Core/Shared/Utilities/ClassificationTypeMap.cs |  2 +-
+ .../AbstractEditAndContinueAnalyzer.cs             |  2 +-
+ .../SolutionCrawlerRegistrationService.cs          |  2 +-
+ .../CommonObjectFormatter.Visitor.cs               |  2 +-
+ .../Utilities/FSharpClassificationTypeMap.cs       |  2 +-
+ .../ExtensionManager/AbstractExtensionManager.cs   |  4 +--
+ .../Core/Portable/Utilities/ReferenceHolder`1.cs   |  4 +--
+ .../SolutionState.CompilationTracker.State.cs      |  2 +-
+ .../Solution/SolutionState.CompilationTracker.cs   |  2 +-
+ .../Solution/SolutionState.UnrootedSymbolSet.cs    |  2 +-
+ .../Remote/TestSerializerService.cs                |  4 +--
+ .../Compiler/Core/ObjectPools/PooledHashSet.cs     |  2 +-
+ 81 files changed, 172 insertions(+), 172 deletions(-)
+
+diff --git a/src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs b/src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs
+index 07c5b16..3522dfa 100644
+--- a/src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs
++++ b/src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs
```

Comment:
This touches too many files to realistically maintain for any amount of time as a patch.